coralproject/talk
# [CORL-1658] AMP Configuration

## What does this PR do?
- Allow AMP support to be enabled inside of `Admin` -> `Config` -> `Advanced`

## How do I test this PR?
- `npm run start:development`
- goto `http://localhost:3000/embed/stream/amp` 
- Check if AMP is really turned on/off
  - It's on when:  `Content-Security-Policy: frame-ancestors` is not sent in the response headers
  - It's off when:  `Content-Security-Policy: frame-ancestors` is sent 

# TODO
- Actual URL to new documentation
